src/System.Security.Permissions/src/System/Security/Permissions/ResourcePermissionBase.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace System.Security.Permissions
 {
+    [Serializable]
```

Comment:
I was wondering if that was the case. I'll push some changes then and we can decide that. Thanks!